.github/workflows/ignored-urls.txt

```diff
@@ -2,3 +2,4 @@ https://doi.org/10.1126/science.aag3009
 https://doi.org/10.1126/science.aah4573
 https://doi.org/10.1002/9780470904640.app3
 https://doi.org/10.xxxx/YYYYYYYY
+https://nextcloud.nfdi4microbiota.de/f/xxxxx
```

Comment:
Thanks for adding the URL here, but this addition will not work as you need to explicitly add all the URLS you want to ignore

```suggestion
https://nextcloud.nfdi4microbiota.de/f/103172
```
as well as the others.

MAybe this URL <https://doi.org/10.xxxx/YYYYYYYY> was misleading in thinking you could it this way, but actually this exact URL is used somewhere in the KB. Could you please add the other URLs?

Reply:
Hi Charlie,

alright, I changed accordingly and thanks for the hint! I thought I was smart, unfortunately, I was not :)